### PR TITLE
test(k8s): remove internet access

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
`Test_ParseVulneDB` calls Internet access.
The result of this test is subject to change.

We need to use test http server for this test.

### CODE REVIEW

**Comment by DmitriyLewen** [comment]
@chen-keinan Hello!

I updated `Test_ParseVulneDB` for `k8s` package.
I removed some CVEs from `k8s-db.json`, but i tried to add CVE for each case from `ParseVulnDBData` and `parseMitreCve` ( i don't check cases from `sanitizedVersions`, because there is `TestSanitizedVersion`).

Take a look these changes, please, when you have time.

File: k8s/k8s_test.go [hunk 1]
```diff
@@ -14,7 +19,26 @@ func Test_ParseVulneDB(t *testing.T) {
...
+		if !d.IsDir() {
+			f, err := os.ReadFile(path)
+			assert.NoError(t, err)
+			mux.HandleFunc(fmt.Sprintf("/%s", filepath.Base(path)), func(w http.ResponseWriter, r *http.Request) {
+				_, err = w.Write(f)
+				assert.NoError(t, err)
+			})
+		}
+		return nil
+	})
```
Comment (chen-keinan) [inline k8s/k8s_test.go:34]: nit: `err error` and `r *http.Request` are not in use 
```suggestion
	err = filepath.WalkDir("./testdata/mitreCVEs", func(path string, d fs.DirEntry, err error) error {
		if !d.IsDir() {
			var f []byte
			f, err = os.ReadFile(path)
			assert.NoError(t, err)
			mux.HandleFunc(fmt.Sprintf("/%s", filepath.Base(path)), func(w http.ResponseWriter, _ *http.Request) {
				_, err = w.Write(f)
				assert.NoError(t, err)
			})
		}
		return nil
	})
```

**Review by chen-keinan — COMMENTED** [review]
lgtm I have added on `nit` comment

File: k8s/k8s_test.go [hunk 2]
```diff
@@ -14,7 +19,27 @@ func Test_ParseVulneDB(t *testing.T) {
...
+	mux := http.NewServeMux()
+	err = filepath.WalkDir("./testdata/mitreCVEs", func(path string, d fs.DirEntry, err error) error {
+		if !d.IsDir() {
+			var f []byte
+			f, err = os.ReadFile(path)
+			assert.NoError(t, err)
+			mux.HandleFunc(fmt.Sprintf("/%s", filepath.Base(path)), func(w http.ResponseWriter, _ *http.Request) {
+				_, err = w.Write(f)
+				assert.NoError(t, err)
+			})
```
Comment (knqyf263) [inline k8s/k8s_test.go:32]: We might be able to leverage `http.FileServer`.
https://pkg.go.dev/net/http#FileServer

If you need to modify the URL's path, `http.StripPrefix` can help.
https://pkg.go.dev/net/http#StripPrefix

**Review by knqyf263 — APPROVED** [review]

File: k8s/k8s_test.go (hunk 2 above)
Comment (DmitriyLewen) [reply k8s/k8s_test.go]: Thanks for the advice!

**Pull request merged**